contrib/hbwin/hbwin.hbp

```diff
@@ -29,7 +29,7 @@ mapi.c
 oemansi.c
 olecore.c
 oleuuid.c{watcom}
-hbolesrv.c
+{HBWIN_HAS_OLESRV}hbolesrv.c
```

Comment:
I don't mind to share it if you want, so you can see if there's is something interesting for the project.